package.json

```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "@js-joda/core": "^3.2.0",
+    "@vuepress/plugin-container": "^2.0.0-beta.27",
```

Comment:
Why was this package added?

Reply:
oh. tried to include a container on the page. It's not required will remove it